<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oranela</class>
 <widget class="QMainWindow" name="oranela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ouroboros</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../imagens/ouroboros.png</normaloff>../../imagens/ouroboros.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="run_bot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Traj">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>84</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Trajetórias</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordenada Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lab_y">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordenada X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lab_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>282</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="Velx">
       <property name="text">
        <string>Velocidade X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Accx">
       <property name="text">
        <string>Aceleração X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Posx">
       <property name="text">
        <string>Posição X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Posy">
       <property name="text">
        <string>Posição Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Accy">
       <property name="text">
        <string>Aceleração Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Vely">
       <property name="text">
        <string>Velocidade Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>981</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Posrel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>84</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Pos rel.</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
